include/dlpack/dlpack.h

```diff
@@ -114,6 +114,26 @@ typedef struct {
   /*! \brief The offset in bytes to the beginning pointer to data */
   uint64_t byte_offset;
 } DLTensor;
+
+/*!
+ * \brief C Tensor object, manage memory of DLTensor. This data structure is
+ *  intended to faciliate the borrowing of DLTensor by another framework. It is
+ *  not meant to transfer the tensor. When the borrowing framework doesn't need
+ *  the tensor, it should call the deleter to handle reference counting.
+ */
+struct DLManagedTensor {
+  /*! \brief DLTensor which is being memory managed */
+  DLTensor dl_tensor;
+  /*! \brief context in which DLManagedTensor is used in a framework. It can
```

Comment:
the context of the original host framework(to emphasize it is the original host) of DLManagedTensor